src/views/chart.js

```diff
@@ -2,7 +2,40 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 
+const initialColaborators = JSON.parse(
+  localStorage.getItem("colaborators") ||
+    JSON.stringify([
+      {
+        id: 0,
+        name: "Carlos Ayala",
+        position: "CEO",
+        parent: "",
+        children: []
```

Comment:
Why did not you put the names of your children here?

Reply:
Oh thanks , I forget about them